compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/chess_instrinsics_wrapper.ll

```diff
@@ -0,0 +1,53 @@
+// This is actually a cpp file that contains a raw string that contains ll
```

Comment:
Could you elaborate in the comment on what kind of stuff this file contains and why it's needed?

Reply:
replace with cpp file stolen from mlir-aie which has an adequate description